ios/Video/Features/RCTVideoUtils.swift

```diff
@@ -165,43 +165,43 @@ enum RCTVideoUtils {
         }
         return nil
     }
-
+    
     static func replaceURLScheme(url: URL, scheme: String?) -> URL? {
         var urlComponents = URLComponents(url: url, resolvingAgainstBaseURL: false)
         urlComponents?.scheme = scheme
-
+        
         return urlComponents?.url
     }
-
+    
     static func extractDataFromCustomSchemeUrl(from url: URL, scheme: String) -> Data? {
         guard url.scheme == scheme,
               let adoptURL = RCTVideoUtils.replaceURLScheme(url:url, scheme: nil) else { return nil }
-
+        
```

Comment:
you insert a lot of tab here 😭

Reply:
Just pressed `ctrl + i` to format code... I think we need swift code formater

Reply:
I agree, but it always better to remove blank at the end of line ...
maybe something can be done in project configuration ?
This blank characters has been removed and added mulitple time, it would really be better not to add them !